cloud-gcp/src/test/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpMetadataCollectorTest.java

```diff
@@ -59,25 +65,21 @@
 
     private static final List<CloudInstance> KNOWN_INSTANCES = Collections.emptyList();
 
-    private static final String PUBLIC_IP = "192.168.1.1";
-
-    private static final String PRIVATE_IP = "10.10.1.1";
-
     @InjectMocks
-    private GcpMetadataCollector underTest;
+    GcpMetadataCollector underTest;
```

Comment:
Why did all these become not private?

Reply:
was at one point a subclass of the new test, not needed now.